includes/polling/isis.inc.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
I would prefer if this were a new style poller module.  See the modules in `LibreNMS/Modules`

Reply:
Changed to use the new style